src/Microsoft.ML.OnnxConverter/OnnxExportExtensions.cs

```diff
@@ -45,5 +67,17 @@ internal static ModelProto ConvertToOnnxProtobuf(this ModelOperationsCatalog cat
         /// <returns>An ONNX model equivalent to the converted ML.NET model.</returns>
         public static void ConvertToOnnx(this ModelOperationsCatalog catalog, ITransformer transform, IDataView inputData, Stream stream) =>
             ConvertToOnnxProtobuf(catalog, transform, inputData).WriteTo(stream);
+
+        /// <summary>
+        /// Convert the specified <see cref="ITransformer"/> to ONNX format and writes to a stream.
+        /// </summary>
+        /// <param name="catalog">The class that <see cref="ConvertToOnnxWithCustomOpSetVersion(ModelOperationsCatalog, ITransformer, IDataView, int, Stream)"/> attached to.</param>
+        /// <param name="transform">The <see cref="ITransformer"/> that will be converted into ONNX format.</param>
+        /// <param name="inputData">The input of the specified transform.</param>
+        /// <param name="opSetVersion"></param>
+        /// <param name="stream">The stream to write the protobuf model to.</param>
+        /// <returns>An ONNX model equivalent to the converted ML.NET model.</returns>
+        public static void ConvertToOnnxWithCustomOpSetVersion(this ModelOperationsCatalog catalog, ITransformer transform, IDataView inputData, int opSetVersion, Stream stream) =>
+            ConvertToOnnxProtobufWithCustomOpSetVersion(catalog, transform, inputData, opSetVersion).WriteTo(stream);
```

Comment:
You are using a different function signature.  Can you keep the same function name? `ConvertToOnnx`